src/components/NcRichContenteditable/NcMentionBubble.vue

```diff
@@ -59,6 +59,10 @@ export default {
 			type: String,
 			required: true,
 		},
+		iconUrl: {
+			type: [String, null],
+			default: null,
+		},
```

Comment:
Very similar component [NcUserBubble](https://nextcloud-vue-components.netlify.app/#/Components/NcUserBubble) has similar prop called `avatarImage`.

Also, `icon` name is usually used for Material Design Icons or CSS `.icon-class` in Nextcloud Vue. Including the `icon` prop of this component - it is a CSS class for an icon too.

I'd consider rename it to either `url` (as in `NcAvatar`) or `avatarUrl` (like in `NcUserBubble`).

Reply:
I used iconUrl, as that is the prop name in userData which is handed over to tribute when autocompleting mention suggestions.
Having to have 2 different properties with the same name or adding another mapping in between felt wrong to me, but yeah I also tripped a bit.

Reply:
Ref https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcRichContenteditable/NcAutoCompleteResult.vue#L66-L73

Reply:
And that refs to https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcRichContenteditable/NcRichContenteditable.vue#L325

Reply:
> And that refs to https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/NcRichContenteditable/NcRichContenteditable.vue#L325

But this is from `linkOptions`, not `autocompleteOptions`. There it is more "icon"

![image](https://user-images.githubusercontent.com/25978914/234586677-f2d191dd-3072-41a6-9e06-ef9a3ddc5c7d.png)

Reply:
Ok, I see now